src/midcom/datamanager/storage/photo.php

```diff
@@ -55,6 +55,9 @@ public function save()
             }
 
             return $this->save_attachment_list();
+        } else if (!empty($this->value['delete'])){
```

Comment:
Please fix coding style (curly brackets should go on a new line, `else if` as well)